src/asm-manual.adoc

```diff
@@ -370,6 +370,48 @@ NOTE: `.option arch, +<ext>, -<ext>` is accepted and will result in enabling the
 extensions that depend on `ext`, e.g. `rv32i` + `.option arch, +v, -v` will result
 `rv32ifd_zve32x_zve32f_zve64x_zve64f_zve64d_zvl32b_zvl64b_zvl128b`.
 
+=== `autocompress`/`noautocompress`
+
+In RISC-V, it is primarily the assembler's job (not the compiler's), to choose
+the shortest possible version of an instruction. This means the assembler will
+change `lw a0, 16(a1)` into `c.lw a0, 16(a1)`, when either the `C` or `Zca`
+extension is enabled.
+
+While this process does save static code size, there are circumstances where it
+might not be wanted, and instead when assembly writers want exactly the
```

Comment:
I think the "when" in "instead when assembly writers" is unnecessary.

Reply:
Done